Subtester/ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2019.4.13f1
-m_EditorVersionWithRevision: 2019.4.13f1 (518737b1de84)
+m_EditorVersion: 2021.2.13f1
```

Comment:
oh nice you updated the Unity version! Great I've been wanting to do this for a while haha. Do you think we have to specify which version of Unity we support (meaning the one we have tested with)? I think we should match Unity IAPs min version

Reply:
That's a great idea! 
They're currently releasing for 2021.3 https://docs.unity3d.com/Manual/com.unity.purchasing.html. 
I'll update to that one in a separate PR